transcom/mymove
# MAIN B-18497 B-18630 closeout office space force fixes

## [B-18497](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18497)
## [B-18630](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18630)

## Related INT PRs
#12256
#12332

## Summary

This PR addresses three issues:
1. Customer's were unable to choose a closeout office for their PPM shipment if they chose the "Space Force" branch of service.
2. Service Counselor's were unable to view/edit a customer's closeout office during counseling if the customer had chosen the "Space Force" branch of service.
3. Closeout office field was absent from shipment card in user's dashboard **after** they had created a shipment but **before** they had signed and sent to SC

All of these should now be possible/fixed.

### How to test

1. Login as a user, create a move with a PPM shipment.
2. On the first screen after choosing "PPM", where you input your estimated weights and departure dates, you should now see a "Closeout Office" field. You can choose whichever you like. Finish creating the shipment.
3.  **Check that the created shipment card contains a "Closeout Office" field with you chosen location.**
4. Complete the move then login as a Service Counselor with a GBLOC corresponding to your move's origin. Find and select the move.
5. **On the shipment card, you should be able to see the customer's chosen closeout office.**
6. If you click Edit Shipment, you should be able to edit the closeout office. Edit this to whatever you like, for this example, I'll edit the office to "PPPO NAVSUP FLCN Annapolis - USN" which has a GBLOC of "BGNC" (different from my move's original GBLOC).
7. Complete counseling of the move, then login as user and complete the "Upload PPM Documents" step to send the move to the closeout queue.
8. Now login as a Service Counselor under the GBLOC of the edited closeout office (mine is "BGNC"), change to the "PPM Closeout" tab, and you should now see your move. You should not see this move under any other GBLOCs.
